polaris-service/build.gradle.kts

```diff
@@ -108,6 +108,12 @@ dependencies {
   testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
 
+if (project.properties.get("eclipseLink") == "true") {
+  dependencies {
+    implementation(project(":polaris-eclipselink"))
```

Comment:
Would it harm to unconditionally include it as a `runtimeOnly` dependency?

For integration-testing, we could also add a separate test-suite to `:polaris-service` that has `:polaris-eclipselink` as well.

Reply:
It's relatively easy to add such a test-suite:
```kotlin
testing {
  suites {
    register("intTestEclipselink", JvmTestSuite::class.java) {
      useJUnitJupiter()

      dependencies {
        implementation(project())
        implementation(project(":polaris-eclipselink") // this would be an implementation-dependency for the suite
      }

      targets {
        all {
          tasks.named("check").configure { dependsOn(testTask) }
        }
      }
    }
}
```

Reply:
I opened https://github.com/polaris-catalog/polaris/discussions/125 to discuss whether we should include the dependency by default